libs/gamify/core/src/index.ts

```diff
@@ -0,0 +1,3 @@
+export * from './lib/gamify-core.module';
```

Comment:
Reference to deleted module file should also be deleted

Reply:
How do you mean? I've renamed the import (export)

Reply:
Well, you kind of deleted the entire file instead of renaming it, don't know if that was on purpose?

Reply:
Good catch! I've renamed the file, but the new name wasn't staged properly. No idea why the CI doesn't complain about it 🤷🏽